File: coderun/1. Median out of three (Средний элемент)/sort/sort.cpp
#include <iostream>
#include <vector>
#include <algorithm>

using namespace std;

int main(int argc, char *argv[])
{
    int a, b, c;
    cin >> a >> b >> c;
    int arr[] = {a, b, c};
    sort(arr, arr + 3);
    cout << arr[1];
    return 0;
}

File: coderun/1. Средний элемент/sort/sort.go
package main

import (
	"fmt"
	"sort"
)

func main() {
	var a, b, c int
	fmt.Scanf("%d %d %d", &a, &b, &c)
	arr := []int{a, b, c}
	sort.Ints(arr)
	fmt.Printf("%d", arr[1])
}
